src/cascadia/TerminalSettingsModel/ActionEntry.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "ActionEntry.h"
+#include "JsonUtils.h"
+
+#include "ActionEntry.g.cpp"
+
+using namespace Microsoft::Terminal::Settings::Model;
+using namespace winrt::Microsoft::Terminal::Settings::Model::implementation;
+
+static constexpr std::string_view ActionIdKey{ "actionId" };

```

Comment:
qq: why should we stutter?

```
{ "type": "action", "actionId": "Terminal.Quit" }
           ^^^^^^    ^^^^^^
                \---/
                  `they are the same word
```

Reply:
hearing arguments on whether it should/could just be "id"

Reply:
FWIW only the json field need change if we decide to call it "id". We don't need to rename the member or anything.

Reply:
aw thanks